doc/sdk-cli-reference.md

```diff
@@ -158,30 +162,61 @@ Create app-operator/.gitignore
 
 ## test
 
-### Flags
+### Available Commands
 
-* `-t, --test-location` **(required)** string - location of e2e test files
-* `-k, --kubeconfig` string - location of kubeconfig for kubernetes cluster
-* `-g, --global-init` string - location of global resource manifest yaml file
-* `-n, --namespaced-init` string - location of namespaced resource manifest yaml file
-* `-f, --go-test-flags` string - extra arguments to pass to `go test` (e.g. -f "-v -parallel=2")
-* `-h, --help` - help for test
+#### local
+Runs the tests locally
 
-### Use
+##### Args
+* <test-location> string - location of e2e test files (e.g. "./test/e2e/")
 
-The operator-sdk test command runs go tests built using the Operator SDK's test framework.
+##### Flags
+* `--kubeconfig` string - location of kubeconfig for kubernetes cluster
+* `--global-manifest` string - location of global resource manifest yaml file
+* `--namespaced-manifest` string - location of namespaced resource manifest yaml file
```

Comment:
Let's clearly specify the defaults for all these flags here. Especially the `--namespaced-manifest` flag which combines the expected files `deploy/sa.yaml`,`deploy/rbac.yaml` and `deploy/operator.yaml`.